packages/service-library/src/servicelib/fastapi/http_client.py

```diff
@@ -1,14 +1,11 @@
 import contextlib
-import logging
 
 import httpx
 from fastapi import FastAPI
 from models_library.healthchecks import IsNonResponsive, IsResponsive, LivenessResult
 
-_logger = logging.getLogger(__name__)
 
-
-class BaseHttpApi:
+class BaseHTTPClient:
```

Comment:
I *intentionally* avoided the word `Client` from the name because in reality this is a **wrapper** of an http client. This naming has lead in the past to situations like:
```python
class MyClient(BaseHTTPClient):
   ...

client = MyClient(..)

# and some lines below something like
foo( url = client.client.url )
# Which client is which now??

```